packages/react/src/components/pickers/BasePicker.tsx

```diff
@@ -361,6 +377,30 @@ export class BasePicker<T extends {}, P extends IBasePickerProps<T>>
     return itemLimit === undefined || items.length < itemLimit;
   }
 
+  protected renderLabel(inputId: string): JSX.Element | null {
+    const { label, disabled, required } = this.props;
+    if (!label) {
+      return null;
+    }
+    return (
+      <Label id={this._id + '-label'} disabled={disabled} required={required} htmlFor={inputId}>
```

Comment:
do we also need to add a way to pass styles through to the label, similar to Textfield's `subComponentStyles`?

I also don't believe the `Label` control needs an `id` here, and I don't see it used anywhere else in the added code. Let me know if I'm missing somewhere it's being used though.